Utilities/bootstrap

```diff
@@ -143,6 +143,12 @@ def add_build_args(parser):
         dest="cross_compile_hosts",
         help="List of cross compile hosts targets.",
         default=[])
+    parser.add_argument(
+        "--cross-compile-flags",
+        help="Swift flags to cross-compile the PackageDescription libraries with CMake")
+    parser.add_argument(
+        "--cross-compile-config",
+        help="Swift flags to cross-compile SPM with itself")
```

Comment:
Even though these two are largely the same, they need to be formatted differently, so [I do that externally and centralize it in `build-script`](https://github.com/apple/swift/pull/33724/files#diff-cc09c8923370b808ebb7439cf4be9d2dde15e81796e80161576ae4cee0502ae7R155).